tests/cli/outbounds.sh

```diff
@@ -6,7 +6,13 @@
 
 FLICT_DIR=$(dirname ${BASH_SOURCE[0]}/)/../../
 
-FLICT="PYTHONPATH=${FLICT_DIR} ${FLICT_DIR}/flict/__main__.py outbound-candidate "
+if [ "$1" != "" ]
```

Comment:
I personally would be more happy if those tests could be rewritten to use `pytest` as the other tests as well

Reply:
There are unit tests, but there are also tests of the cli interface to make sure the cli is the same (and working).

Reply:
and even that could be done with pytest :-)
the alternative would be to enable this shell script based testing in CI as well (which isn't the case atm) - I just want to make sure we aren't degrading in quality in the long run

Reply:
I get the impression you're trying to tell me something:)